.github/workflows/first-actions.yml

```
@@ -1,6 +1,6 @@
 name: My First GitHub Actions
 
-on: [push]
+on: [pull]
```

actionlint (1.7.7)

unknown Webhook event "pull". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)